src/main/java/Food

```diff
@@ -0,0 +1,52 @@
+/**
+ * This class stores a Food Object from the database,
+ * describing its nutrient level
+ */
+
+public class Food {
+
+    public double calories;
+    public double carbohydrate;
+    public double proteins;
+    public double fat;
+    public double sugar;
+
+    public String foodType;
+    public String id;
+    public boolean vegFriendly;
+    public int nutrientScore;
+
+    /**
+     *
+     * @param calories is a percentage of recommended c
```

Comment:
this sentence doesn't seem to be finished?

Reply:
Yepyep sorry, fixed!